test/system/500-networking.bats

```diff
@@ -162,6 +162,24 @@ load helpers
     done
 }
 
+@test "podman run with slirp4ns assigns correct gateway address to host.containers.internal" {
+    run_podman run --network slirp4netns:cidr=192.168.0.0/24 \
```

Comment:
Just curious: what problem does `random_rfc1918_subnet()` have that prevents you from using it?

Reply:
I wasn't aware of that function. I added it to the tests I wrote :+1:

Reply:
Thanks! One suggestion, though: I find string concatenation easier to read than stripping. Would you consider:
```
    subnet=$(random_rfc1918_subnet)
    ... :cidr="$subnet.0/24"
    .... grep "$subnet"
    ... nameserger "${subnet}.3"
```
That removes the need for all those complicated fragile `%.*`, and makes life much easier for future maintainers.

Reply:
Sounds good. Fixed: c8dfcce6db0adfa2f91d979271e8c2725eb753a6